tox.ini

```diff
@@ -6,18 +6,20 @@ setenv =
     DJANGO_SETTINGS_MODULE=tests.settings
     PYTHONPATH = {toxinidir}:{toxinidir}/omni_blocks
 commands =
-    coverage run --source omni_blocks runtests.py
-    coverage report --show-missing --omit=.tox/*,*/tests/*,
+    pytest
```

Comment:
Is `pytest` giving us coverage stats?

Reply:
No it is not, I will update the tox files in each of the libraries to run:
```
coverage run --source='.' -m pytest
coverage report --show-missing --omit=*migrations*,*tests*,.tox*
```
(Taken from 1610 travis file)